lib/hg-utils.coffee

```diff
@@ -410,6 +414,21 @@ class Repository
       @handleHgError(error)
       return null
 
+  # This checks to see if the current params indicate whether we are working
+  # with the current repository.
+  #
+  # * `params` The params that are going to be sent to the hg command {Array}
+  #
+  # Returns a {Boolean} indicating if the rootPath was found in the params
+  isCommandForRepo: (params) ->
+    rootPath = @rootPath
+
+    paths = params.filter (param) ->
+      normalizedPath = path.normalize((param || ''))
```

Comment:
Sometimes I had got paths starting with `../`. I'm removing this characters somewhere but I'm not sure if it will cause any problem... I guess no. If I find any problem I'll let you know.